concepticondata/conceptlists/Luangthongkum-2019-341.tsv

```diff
@@ -28,13 +28,13 @@ Luangthongkum-2019-341-26	26	blow (a fire)	176	BLOW (WITH MOUTH)
 Luangthongkum-2019-341-27	27	boar (wild~)	3154	WILD BOAR
 Luangthongkum-2019-341-28	28	boat	1844	BOAT
 Luangthongkum-2019-341-29	29	body dirt		
-Luangthongkum-2019-341-30	30	boil (v.)	2493	BOIL
+Luangthongkum-2019-341-30	30	boil (v.)	2492	BOIL (SOMETHING)
```

Comment:
Its not specified whether it is the transitive or intransitive meaning, so we should stay with the broader mapping BOIL